test_work/run_mig.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
We can move it, but I'm not sure I'd use script.  These probably are most related to wagn, so wagn/test/script might be good.  What do you think?

There isn't much of a dependency on the script location, but where the working directory goes is pretty important.  The script needs to know where it (the working dir, not the script dir) is relative to the repo-root, so it would be good if we all put it at the same level in the dir hierarchy.